docs/user/intelmq-api.rst

```diff
@@ -199,7 +199,7 @@ Can I just install it from the deb/rpm packages while installing IntelMQ from a
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Yes, you can install the API and the Manager from the deb/rpm repositories, and install your IntelMQ from a somewhere else, e.g. a local repository.
-However, knowledge about Python system administration experience and is recommended if you do so.
+However, knowledge about Python system administration experience is recommended if you do so.
```

Comment:
That `and` was probably at the wrong place and it should be `Python **and** system administration experience`